db/migrate/20200423070119_create_categories.rb

```diff
@@ -0,0 +1,9 @@
+class CreateCategories < ActiveRecord::Migration[6.0]
+  def change
+    create_table :categories do |t|
+      t.string :name
```

Comment:
`null: false`
I seems be strange if in database will be category with empty name